server/datastore/mysql/maintained_apps_test.go

```diff
@@ -267,6 +288,37 @@ func testListAndGetAvailableApps(t *testing.T, ds *Datastore) {
 	maintained1.TitleID = ptr.Uint(titleID)
 	require.Equal(t, maintained1, gotApp)
 
+	// we haven't added the windows app yet, so we shouldn't have a title ID for it
+	apps, meta, err = ds.ListAvailableFleetMaintainedApps(ctx, &team1.ID, fleet.ListOptions{IncludeMetadata: true})
+	require.NoError(t, err)
+	require.Len(t, apps, 4)
+	require.EqualValues(t, meta.TotalResults, 4)
+	require.Nil(t, apps[3].TitleID)
+
+	// add windows app
+	_, windowsTitleID, err := ds.MatchOrCreateSoftwareInstaller(ctx, &fleet.UploadSoftwareInstallerPayload{
+		Title:           "Maintained4",
```

Comment:
```suggestion
		Title:           "Maintained4 (MSI)",
```

For this to work, it'll have to be rebased off of the bugfix I just submitted.